README.md

```diff
@@ -98,12 +98,13 @@ Add in the following references:
 
 1. Microsoft Scripting Runtime
 2. Microsoft Visual Basic for Applications Extensibility 5.3
+3. Windows Script Host Object Model
+4. Microsoft Shell Controls And Automation
```

Comment:
Need to add these to the CodeExport.config.json for those who are using CodeExport to automatically build it.